doc/releases/eol_releases.rst

```diff
@@ -1,7 +1,15 @@
+.. _eol_releases:
+
 End-of-life releases
-====================
+####################
+
+Release notes and migration guides for end-of-life releases of Zephyr RTOS are kept here for
+historical purposes.
+
+.. _eol_releases_relesase_notes:
 
-Release notes for end-of-life releases of Zephyr RTOS are kept here for historical purposes.
+Release Notes
```

Comment:
```suggestion
Release notes
```

Reply:
ditto

Reply:
Then you might want to align the capitalization of `End-of-life releases` at the beginning of this page.